google/resource_bigquery_table_test.go

```diff
@@ -221,6 +221,83 @@ func TestAccBigQueryTable_updateView(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryTable_MaterializedView_DailyTimePartioning_Basic(t *testing.T) {
+	t.Parallel()
+
+	datasetID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	tableID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	materialized_viewID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	query := fmt.Sprintf("SELECT count(some_string) as count, some_int, ts FROM `%s.%s` WHERE DATE(ts) = '2019-01-01' GROUP BY some_int, ts", datasetID, tableID)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccBigQueryTableWithMatViewDailyTimePartitioning_basic(datasetID, tableID, materialized_viewID, query),
+			},
+			{
+				ResourceName:      "google_bigquery_table.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				ResourceName:      "google_bigquery_table.mv_test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccBigQueryTable_MaterializedView_DailyTimePartioning_Update(t *testing.T) {
+	t.Parallel()
+
+	datasetID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	tableID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+	materialized_viewID := fmt.Sprintf("tf_test_%s", randString(t, 10))
+
+	query := fmt.Sprintf("SELECT count(some_string) as count, some_int, ts FROM `%s.%s` WHERE DATE(ts) = '2019-01-01' GROUP BY some_int, ts", datasetID, tableID)
+
+	enable_refresh := "false"
+	refresh_interval_ms := "3600000"
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigQueryTableDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
I'm seeing test failures sometimes due to a couple of fields marked `Computed` automatically updating. Specifically `etag` and `last_modified_time`. These are computed and should not matter, as they change outside of Terraform's state management.

We need to specifically ignore these during the import test to prevent failures. Can you add `ImportStateVerifyIgnore: []string{"etag", "last_modified_time"},` to the import steps in these tests?